File: Code/Web Development/eCoaching/Models/Survey/Answer.vb
<Serializable()> Public Class Answer
    Public Property ID As Integer
    Public Property SingleChoice As Integer
    Public Property MultiLineText As String
    Public Property QuestionID As Integer

    Sub New()
        ID = -1
        SingleChoice = -1
        MultiLineText = String.Empty
        QuestionID = -1
    End Sub
End Class

File: Code/Web Development/eCoachingLog/eCoachingLog/Models/Survey/Answer.cs
namespace eCoachingLog.Models.Survey
{
	public class Answer
	{
		public int Id { get; set; }
		public int SingleChoiceSelected { get; set; } 
		public string MultiLineText { get; set; }
		public int QuestionId { get; set; }

		public Answer()
		{
			this.Id = -1;
			this.SingleChoiceSelected = -1;
			this.MultiLineText = string.Empty;
			this.QuestionId = -1;
		}
	}
}